projects/klippa/ngx-enhancy-forms/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klippa/ngx-enhancy-forms",
-  "version": "14.2.1",
+  "version": "14.2.2",
```

Comment:
when you add a new feature, we update the minor version. This is patch, and patch is for bugfixes.
so it will be 14.3.0

Reply:
Fixed @wouter-willems ! Thanks